docs/AppExtensions.md

```diff
@@ -0,0 +1,27 @@
+---
+id: app-extensions
+title: App Extensions
```

Comment:
Will probably just change to this! That way I don't have to talk about other app extensions, which I haven't really tried out